cookiecutter/{{cookiecutter.organization}}-richie-site-factory/bin/activate

```diff
@@ -38,11 +43,5 @@ if [[ ${choice} -le 0 ]]; then
     choice=${default}
 fi
 
-RICHIE_SITE="${sites[$((choice-1))]}"
-
-if [[ "${BASH_SOURCE[0]}" != "${0}" ]] ; then
-    export RICHIE_SITE
-else
-    echo -e "\\n# Copy/paste ${RICHIE_SITE} activation command:"
-    echo "export RICHIE_SITE=${RICHIE_SITE}"
-fi
+# Save activated site to file
+echo "${sites[$((choice-1))]}" > .site
```

Comment:
From my point of view, I seems legit to put this file into env.d folder. WDYT?
```suggestion
echo "${sites[$((choice-1))]}" > env.d/site
```

Reply:
Good idea :pray: